databricks/koalas/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing to Koalas - design and principles
+
+This document gives guidance to developers if they plan to contribute to Koalas.
+In particular, it answers the questions:
+ - what is in the scope of the Koalas project? What should go into PySpark or Pandas instead?
+ - What is expected for code contributions
+
+Koalas helps developers familiar with the Pandas API to scale their data science pipelines with Spark.
+As such, it focuses on existing users of pandas.
+ - it focuses on the high level, public APIs that users are usually working with
+ - it respects to the largest extent the conventions of the Python numerical ecosystem, and allows the use of numpy types, etc. that are supported by Spark.
+ - it should clearly document when results may differ between Spark and pandas (which may happen because Spark rows are not ordered)
+
+
+### A unified API for data science
+
+The Koalas dataframe is meant to provide the best of pandas and Spark under a single API, with easy and clear conversions 
+between each API when necessary. It aims at incorporating:
+ - most of the data transform tools from pandas
```

Comment:
I think if we add Spark SQL methods with Pandas there will be duplicate ones ! 
Example : `pd.DataFrame.where()` and spark `where()`

Also, it will be a little bit confusing for users that familier only with Pandas.